spec/features/issues_spec.rb

```diff
@@ -172,6 +172,31 @@
             expect(page).to have_content("treść rezenzji")
           end
         end
+        context "recenzenci" do
+          before do
+            Person.create!(name: "Andrzej", surname: "Kapusta", email: "a.kapusa@gmail.com", sex:
+                           "mężczyzna", roles: ['redaktor', 'recenzent'])
+            article_file = Rails.root.join("spec/features/files/plik.pdf").open
+            revision = ArticleRevision.create!(submission: Submission.first, pages: 1, pictures: 1, version: 1, article: article_file, received: "18-01-2016")
+            Review.create!(article_revision: revision, deadline: '28/01/2016', person: Person.first,
+                           status: "recenzja pozytywna", asked: '1/01/2016', content: "treść rezenzji")
+          end
+
+          scenario "wyświetlenie linku do listy recenzentów" do
+            visit "/issues"
+
+            click_link "4"
+            expect(page).to have_link("Pokaż recenzentów")
+          end
+
+          scenario "wyświetlenie listy recenzentów" do
+            visit "/issues"
+
+            click_link "4"
```

Comment:
Pojawił się jakiś niezrozumiały przeze mnie błąd. Zrobiłam pulla z apohllo
matera i migrację bazy danych. Być może jest to spowodowane zrobienie
commitu, a następnie wprowadzeniem nowych zmian i zrobieniem kolejnego
commitu i dopiero jego wysłanie do głównej gałęzi. Nie wiem czy
wcześniejszy commit robi konflikt, czy znika.

! [rejected]        26-generowanie-listy-recenzentow ->
26-generowanie-listy-recenzentow (fetch first)
error: failed to push some refs to 'https://github.com/mbasik/rocznik.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.


2016-02-26 12:36 GMT+01:00 Aleksander Smywiński-Pohl <
notifications@github.com>:

> In spec/features/issues_spec.rb
> <https://github.com/apohllo/rocznik/pull/383#discussion_r54233181>:
>
> > +            revision = ArticleRevision.create!(submission: Submission.first, pages: 1, pictures: 1, version: 1, article: article_file, received: "18-01-2016")
> > +            Review.create!(article_revision: revision, deadline: '28/01/2016', person: Person.first,
> > +                           status: "recenzja pozytywna", asked: '1/01/2016', content: "treść rezenzji")
> > +          end
> > +
> > +          scenario "wyświetlenie linku do listy recenzentów" do
> > +            visit "/issues"
> > +
> > +            click_link "4"
> > +            expect(page).to have_link("Pokaż recenzentów")
> > +          end
> > +
> > +          scenario "wyświetlenie listy recenzentów" do
> > +            visit "/issues"
> > +
> > +            click_link "4"
>
> jw.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apohllo/rocznik/pull/383/files#r54233181>.
>